src/core/Core.js

```diff
@@ -619,11 +626,57 @@ class Uppy {
     }
   }
 
-/**
- * Logs stuff to console, only if `debug` is set to true. Silent in production.
- *
- * @return {String|Object} to log
- */
+  /**
+  * Set info message in `state.info`, so that UI plugins like `Informer`
+  * can display the message
+  *
+  * @param {string} msg Message to be displayed by the informer
+  */
+
+  info (msg, type, duration) {
+    this.setState({
+      info: {
+        isHidden: false,
+        type: type,
+        msg: msg
+      }
+    })
+
+    this.emit('core:info-visible')
+
+    window.clearTimeout(this.info.timeoutID)
```

Comment:
Suuuper tiny nit, but `this.info.timeoutID` confused me first (I thought it was supposed to be `this.state.info.timeoutID`). Could this be a separate property like `this.infoTimeoutID` instead?